start-spiget

```diff
@@ -26,32 +27,67 @@ getResourceFromSpiget() {
 
   log "Downloading resource ${resource} ..."
 
-  mkdir -p /data/plugins
+  if [ ! -d /data/plugins ]; then
+    mkdir -p /data/plugins
+  fi
 
-  if [ -f /data/plugins/.${resource} ]; then
-    log "Resource '${$resource}' already downloaded"
-  else
-    tmpfile="/tmp/${resource}.zip"
-    url="https://api.spiget.org/v2/resources/${resource}/download"
-    if ! curl -o "${tmpfile}" -fsSL -H "User-Agent: itzg/minecraft-server" "${extraCurlArgs[@]}" "${url}"; then
-      log "ERROR failed to download resource '${resource}' from ${url}"
-      exit 2
-    fi
+  versionfile="/data/plugins/.${resource}-version.json"
+  versionfileNew="/tmp/.${resource}-version.json"
+
+  if [ -f "$versionfile" ]; then
+    versionfileTimestamp=$(date +%s -r $versionfile)
+    versionfileMinTimestamp=$(($(date +%s) - 60 * $SPIGET_DOWNLOAD_TOLERANCE))
```

Comment:
Double check me on this, but I think you can simplify this and the condition below with a `find` trick like:

```
if [[ -n $(find "$versionfile" -mmin +${SPIGET_DOWNLOAD_TOLERANCE}) ]]; then
  urlVersion="https://api.spiget.org/v2/resources/${resource}/versions/latest"
  # and so on
```